keyboards/lily58/keymaps/nazu/config.h

```diff
@@ -0,0 +1,121 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2020 Ben Roesner (keycapsss.com)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Select hand configuration */
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#ifdef RGBLIGHT_ENABLE
+#   define RGBLIGHT_ANIMATIONS
+#   define RGBLIGHT_HUE_STEP 6 // number of steps to cycle through the hue by
+#   define RGBLIGHT_SAT_STEP 6 // number of steps to increment the saturation by
+#   define RGBLIGHT_VAL_STEP 6 // number of steps to increment the brightness by
+#   define RGBLIGHT_SLEEP //  the RGB lighting will be switched off when the host goes to sleep
+#   define RGBLIGHT_SPLIT // synchronization functionality for split keyboards
+#   undef RGBLIGHT_ANIMATIONS // disable all RGBLIGHT effects
```

Comment:
`RGBLIGHT_ANIMATIONS` is deprecated. You may need to undef the specific animation modes, now, though. 
```suggestion
```